statistics/histogram.go

```diff
@@ -205,6 +204,11 @@ func IsAnalyzed(flag int64) bool {
 	return (flag & AnalyzeFlag) > 0
 }
 
+// ResetAnalyzeFlag resets the AnalyzeFlag.
```

Comment:
After this change, `AnalyzeFlag` means that the statistics comes from the `analyze` command, and has not been modified by the query feedback?